src/tools/login.js

```diff
@@ -0,0 +1,17 @@
+export function loginToServer() {
+  const url = '/oauth/login/github'
+  const name = 'github_login'
+  const specs = 'width=500,height=600'
+  const authWindow = window.open(url, name, specs)
+  authWindow.focus()
+}
+
+export function logoutFromServer(dispatch, successCallback, errorCallback) {
```

Comment:
I would make `dispatch` the last parameter, that way you can just omit it in cases where it does not apply (i.e. `user-menu.jsx`)